generate-csv.js

```diff
@@ -115,18 +134,13 @@ function writeToGnosisCSV(filePathGnosis, csvRows) {
 function writeToParcelCSV(filePathParcel, csvRows) {
     console.log('writeToParcelCSV')
 
-    // Add missing column
-    csvRows.forEach(function(row, index) {
-        row.token_address = '0x333A4823466879eeF910A04D473505da62142069'
-    })
-    
     const writer = csvWriter.createObjectCsvWriter({
         path: filePathParcel,
         header: [
             {id: 'name', title: 'Name(Optional)'},
-            {id: 'receiver', title: 'Address/ENS'},
-            {id: 'amount', title: 'Amount'},
-            {id: 'token_address', title: 'Token Address/Token Symbol'},
+            {id: 'receiver', title: 'receiver'},
```

Comment:
Is this this code changed needed for removing zero amount rows?

Reply:
Messed up merge, fixed!